src/main/java/com/couchbase/lite/replicator/PullerInternal.java

```diff
@@ -845,63 +845,45 @@ public void run() {
 
     @Override
     public void changeTrackerCaughtUp() {
-        workExecutor.submit(new Runnable() {
-            @Override
-            public void run() {
-                try {
-                    Log.d(Log.TAG_SYNC, "changeTrackerCaughtUp");
-                } catch (Exception e) {
-                    e.printStackTrace();
-                    throw new RuntimeException(e);
-                }
-            }
-        });
-
+        Log.e(Log.TAG_SYNC, "changeTrackerCaughtUp");
```

Comment:
Should this be something else other than Log.e? I understand that Log.e is for error.